src/app/views/communities/view/overview/edit/games/games.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+	<div>
+		<app-community-perms :community="community" required="community-channels">
+			<h2 class="section-header">
+				<translate>Linked Games</translate>
+			</h2>
+
+			<div class="page-help">
+				<p>
+					<translate>
+						Link your games to your community to do something something more flavor text here.
+						Probably something about how the community appears on the game page, and the game
+						appears in the sidebar. Also maybe that hidden games don't appear in the sidebar, but
+						the community does appear on the game page
+					</translate>
+				</p>
+			</div>
+
+			<div v-if="canLinkNewGames">
+				<app-button block primary @click="onClickLinkGame">
+					<translate>Link Game</translate>
+				</app-button>
+			</div>
+			<div v-else>
+				<p>
+					<translate :translate-params="{ count: linkGameCount }">
+						Probably some text about how you can only link %{ count } games; or that there are no
+						more games left to link.
+					</translate>
```

Comment:
using translation params in a placeholder text made me laugh and i don't know why